autoscheduler/frontend/src/components/SchedulingPage/OptionsCard/OptionsCard.css

```
@@ -0,0 +1,10 @@
+.button-container {
+    display: flex;
+    flex-direction: column;
+    padding: 8px;
+    justify-content: space-between;
```

```suggestion
    justify-content: space-between;
    user-select: none;
```
I don't think we should enable text highlighting on components that will be repeatedly clicked (like buttons) since it looks bad when it happens